python/smqtk/web/search_app/modules/iqr/static/js/smqtk.data_view.js

```diff
@@ -18,7 +18,7 @@ function DataView(container, rank, uid, probability, saliency_flag, is_example,
     this.uid = uid;
     this.probability = probability;
     this.saliency_flag = saliency_flag;
-    this.is_example = is_example === undefined ? false : is_example;
+    this.is_example =  is_example;
```

Comment:
Was this change required?

Reply:
No, was a result of something i wanted to try. But changed it to the previous version now